### ISSUE TITLE
Random tweaks/extensions not working - dashboard, search, and likes

### ISSUE THREAD

**Issue by commodorecliche** [new-xkit/XKit#1907]
Random tweaks and extensions are not working, while others continue to work just fine. Ex: I cannot get rid of the like animation, despite selecting the dashboard tweak. The search has reverted back to tumblr's default rather than classic. Can no longer search likes by page. I imagine tumblr has updated and done something stupid that's interfering with xkit.

**Comment by vexx32** [comment]
Yeah, lots of stuff got broken over the update. See #1802 for a summary of progress on each extension. 🙂

**Comment by commodorecliche** [comment]
dang that is quite a hefty list of screwiness. awesome to see the progress, thanks for the link.
